src/java/org/apache/cassandra/utils/btree/BTree.java

```diff
@@ -4186,12 +4040,6 @@ int init(Object[] tree)
             return leafDepth + 1;
         }
 
-        void advanceLeaf()
-        {
-            if (++positions[depth] == sizeOfLeaf(nodes[depth]))
-                ascendToUnfinishedParent();
```

Comment:
`ascendToUnfinishedParent` is now unused

Reply:
Removed